adresse/static/main.css

```diff
@@ -392,6 +413,32 @@ footer .twitter:before {
     left: -31px;
     float: right;
 }
+.flashes li {
+    padding: 10px;
+    text-align: center;
+}
+.flashes li:before,
+.flashes li:after {
+    font-family: 'open_sansextrabold';
+}
+.flashes li.success {
```

Comment:
If you care about performances, it's probably better to remove the `li` selector here and below.